KeeTrayTOTP/Menu/TrayMenuItemProvider.cs

```diff
@@ -96,10 +94,9 @@ private ToolStripMenuItem CreateDatabaseMenuItemForDocument(PwDocument document)
             {
                 var documentName = UrlUtil.GetFileName(document.Database.IOConnectionInfo.Path);
                 mainDropDownItem = new ToolStripMenuItemEx(documentName, ImageExtensions.CreateImageFromColor(document.Database.Color));
+                mainDropDownItem.ForceDropDownArrow = true;
```

Comment:
If we're always using this with true, maybe move this to the ctor in ToolStripMenuItemEx

Reply:
At this line we are using the Control with ``false`` behavior:
https://github.com/KeeTrayTOTP/KeeTrayTOTP/blob/b767d9dd8231ac684a9a56ce026469ebdbb3243a/KeeTrayTOTP/Menu/TrayMenuItemProvider.cs#L91

I also don't really like the idea that the behavior is turned on by default.

Reply:
Ah, didn't notice that one. That's fair.